lambdas/service/app.py

```diff
@@ -688,3 +693,68 @@ def _file_url(uuid, **kwargs):
     return (config.service_endpoint() + '/dss/files' +
             '/' + urllib.parse.quote(uuid, safe='') +
             '?' + urllib.parse.urlencode(kwargs))
+
+
+@app.route('/query', methods=['POST'], cors=True)
```

Comment:
I don't feel great about this path so any suggestions would be appreciated.  `query` kind of makes sense to me because I see it as creating a query resource.  `/resources/query`?

Reply:
Let's just do`/urls`.